src/app/modules/user/user.middleware.ts

```diff
@@ -26,8 +28,9 @@ export const validateContactOtpVerificationParams = celebrate({
   }),
 })
 
-export const validateUpdateUserLastSeenFeatureUpdateVersion = celebrate({
+export const validateUpdateUserLastSeenFlagVersion = celebrate({
   [Segments.BODY]: Joi.object({
     version: Joi.number().required(),
+    flag: Joi.string().valid(...Object.values(SeenFlags)),
```

Comment:
Should this be `required()` as well?

```suggestion
    flag: Joi.string().valid(...Object.values(SeenFlags)).required(),
```

Reply:
Good catch! d883952